azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/middleware/sdk/ChatServiceListener.kt

```diff
@@ -101,10 +112,17 @@ internal class ChatServiceListener(
             is ParticipantTimestampInfoModel -> {
                 when (it.eventType) {
                     ChatEventType.TYPING_INDICATOR_RECEIVED -> {
-                        dispatch(ParticipantAction.TypingIndicatorReceived(message = it.infoModel))
+                        val infoModel = it.infoModel
+                        dispatch(ParticipantAction.AddParticipantTyping(infoModel = infoModel))
+                        coroutineScope.launch {
```

Comment:
do we need logic here to cancel this scope if a new Typing indicator comes in for the same participant?

Reply:
Also, Magic number, break out to a constant or val.

Reply:
We can do in future to improve performance. The notification will do nothing as participant does not exist